dev/devicelab/lib/tasks/web_benchmarks.dart

```diff
@@ -216,6 +295,17 @@ Future<TaskResult> runWebBenchmark(WebBenchmarkOptions benchmarkOptions) async {
     } finally {
       unawaited(server.close());
       chrome?.stop();
+      if (flutterRunProcess != null) {
+        // Sending a SIGINT/SIGTERM to the process here isn't reliable because [process] is
+        // the shell (flutter is a shell script) and doesn't pass the signal on.
+        // Sending a `q` is an instruction to quit using the console runner.
+        flutterRunProcess.stdin.write('q');
+        await flutterRunProcess.stdin.flush();
+        // Give the process a couple of seconds to exit and run shutdown hooks
+        // before sending kill signal.
+        await Future<void>.delayed(const Duration(seconds: 2));
```

Comment:
Instead of unconditionally waiting and killing every time, instead we could do like:
```
await flutterRunProcess.exitCode.timeout(const Duration(seconds: 2), onTimeout: () => flutterRunProcess.kill(io.ProcessSignal.sigint));
```
Or something like that.

Reply:
Sounds good - added! Also added some Chrome flags that'll let the process run backgrounded.